Various technical notes/transition_infos.json

```
@@ -379,13 +374,7 @@
       "area_id": "0xB8D5CE86",
       "area_name": "Mouth of Inti",
       "default_entrance": "0x70EBFCA3",
-      "exits": [
-        {
-          "area_id": "0x70EBFCA3",
-          "area_name": "Altar of Huitaca (2 exits)",
-          "requires": []
-        }
-      ]
+      "exits": []
```

Boss fight does lead back to Altar

```suggestion
      "exits": [
        {
          "area_id": "0x70EBFCA3",
          "area_name": "Altar of Huitaca (2 exits)",
          "requires": []
        }
      ]
```

---

As mentioned in both the commit titles as well as the utils.py file that writes it to the Spoiler Log:
**Mouth of Inti** is temporarily removed from the randomization proccess. An essential part of that removal is that this exit is not included in the .json file.